clang/test/dpct/sharedmem_var_static.cu

```diff
@@ -82,7 +82,13 @@ void testTemplate() {
 
   // CHECK: dpct::get_default_queue().submit(
   // CHECK-NEXT:   [&](sycl::handler &cgh) {
+  // CHECK-NEXT:     /*
+  // CHECK-NEXT:     DPCT1099:{{[0-9]+}}: 'size * 4' expression was replaced with a value. Modify the code to use original expression, provided in comments, if it is correct.
+  // CHECK-NEXT:     */
   // CHECK-NEXT:     sycl::local_accessor<T, 2> s_acc_ct1(sycl::range<2>(64/*size * 2*/, 128/*size * 4*/), cgh);
```

Comment:
There were 2 different expressions replaced, but only one warning. I think, there should be 2 warnings or one warning, that would list both cases.

Reply:
my mistake, fixed.

Reply:
If you use 1 warning for several expressions, the wording needs to be updated. See my comment above with changes required. The point is, this is not one expression, these are several expressions.